lib/publiccloud/utils.pm

```diff
@@ -342,27 +342,22 @@ Transactional systems like SLE micro shall use C<transactional_update patch> and
 
 sub fully_update_system {
```

Comment:
```suggestion
sub ssh_update_transactional_system {
```

Reply:
That name came out during a meeting call, but I agree renaming it now